main.py

```diff
@@ -0,0 +1,75 @@
+import os
+
+from agent import get_graph
+from inputs import from_github
+from langchain_core.messages import HumanMessage
+from tools import get_pr_metadata
+
+from composio import Action
+
+import logging
+logging.getLogger().setLevel(logging.DEBUG)
+
+# TODO: 
+# 1. API_KEY
```

Comment:
Ensure that sensitive information such as API keys are not hardcoded in the scripts and are instead fetched from environment variables or secure storage.